src/sequelize-typescript.ts

```diff
@@ -292,7 +292,7 @@ export abstract class SequelizeTypeScript {
       return result;
     };
 
-    const als = this.#transactionAls;
+    const als = this.#transactionCls;
```

Comment:
cls*

Reply:
This happens a few other times in this file as well

Reply:
Done